### PR TITLE
CCFRI-3971 - Change Notification Form document upload bug fix

### PR DESCRIPTION
**Pull request by vietle-cgi**
# Description

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/requestChanges/ChangeNotificationForm.vue [hunk 1]
```diff
@@ -97,25 +94,26 @@
...
+import { CHANGE_TYPES, DOCUMENT_TYPES } from '@/utils/constants.js';
 
-import NavButton from '../../components/util/NavButton.vue';
-import ChangeFileUpload from './ChangeFileUpload.vue';
+import AppButton from '@/components/guiComponents/AppButton.vue';
+import NavButton from '@/components/util/NavButton.vue';
+import ChangeFileUpload from '@/components/requestChanges/ChangeFileUpload.vue';
 
 export default {
   name: 'ReportChange',
```
Comment (weskubo-cgi) [inline frontend/src/components/requestChanges/ChangeNotificationForm.vue:112]: I noticed this component and ChangeInformation both have 'ReportChange' copied as the name.

File: frontend/src/components/summary/changeRequest/ChangeNotificationFormSummary.vue [hunk 2]
```diff
@@ -110,10 +110,14 @@ export default {
       );
     },
     supportingDocuments() {
-      return this.changeNotificationFormDocuments?.filter((document) => document.subject == 'SUPPORTING_DOC');
+      return this.changeNotificationFormDocuments?.filter(
+        (document) => document.subject == DOCUMENT_TYPES.CR_NOTIFICATION_FORM_SUPPORTING,
```
Comment (weskubo-cgi) [inline frontend/src/components/summary/changeRequest/ChangeNotificationFormSummary.vue:114]: Should this be ===? Same for below.

File: frontend/src/store/reportChanges.js [hunk 3]
```diff
@@ -1,14 +1,13 @@
 import { isEmpty } from 'lodash';
```
Comment (weskubo-cgi) [inline frontend/src/store/reportChanges.js:1]: Can we kill the non error related console logs in this file?

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/requestChanges/ChangeNotificationForm.vue (hunk 1 above)
Comment (vietle-cgi) [reply frontend/src/components/requestChanges/ChangeNotificationForm.vue]: it's updated

File: frontend/src/components/summary/changeRequest/ChangeNotificationFormSummary.vue (hunk 2 above)
Comment (vietle-cgi) [reply frontend/src/components/summary/changeRequest/ChangeNotificationFormSummary.vue]: it's updated

File: frontend/src/store/reportChanges.js (hunk 3 above)
Comment (vietle-cgi) [reply frontend/src/store/reportChanges.js:1]: it's updated

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**